libs/components/layout/src/workspace-sidebar/page-tree/tree-item/styles.ts

```diff
@@ -182,8 +184,7 @@ export const TextLink = styled(Link, {
     appearance: none;
     text-decoration: none;
     user-select: none;
-    color: ${({ theme, active }) =>
-        active ? theme.affine.palette.primary : 'unset'};
+    color: #4c6275;
```

Comment:
use `theme.affine.palette.secondaryText`

Reply:
next pr adjust it😄